dot/rpc/modules/author_test.go

```diff
@@ -51,12 +53,12 @@ func TestAuthorModule_HasSessionKey_WhenScaleDataEmptyOrNil(t *testing.T) {
 	require.NoError(t, err)
 	require.False(t, bool(res))
 
-	coremockapi.AssertCalled(t, "DecodeSessionKeys", mock.AnythingOfType("[]uint8"))
+	coremockapi.AssertCalled(t, "DecodeSessionKeys", common.MustHexToBytes("0x0400"))
 }
 
 func TestAuthorModule_HasSessionKey_WhenRuntimeFails(t *testing.T) {
```

Comment:
Can we revise all the `HasSessionKey` to follow the slice of anonymous structs style of tests.